core/src/main/java/hudson/model/Computer.java

```diff
@@ -778,15 +778,15 @@ public String getUrl() {
     @Exported
     public Set<LabelAtom> getAssignedLabels() {
         Node node = getNode();
-        return (node != null) ? node.getAssignedLabels() : Collections.EMPTY_SET;
+        return (node != null) ? node.getAssignedLabels() : Collections.emptySet();
```

Comment:
IIRC it reverts the recent change by @linuxsuren

Reply:
Is that so? According to github's blame it seems that this hasn't been changed in 5 years. I changed this as I was in this bit of code and it was warning on my IDE